### ISSUE TITLE
Detect when animation is finished

### ISSUE THREAD

**Issue by erperejildo** [Klerith/animate_do_package#30]
Any available listener for this?

**Comment by Klerith** [comment]
Interesting, I will explore the possibility of exposing a listener here; in the meantime, if you need this, you can use the exposed controller; however, that will require the manual trigger to launch the animation.

Let me work on this now.

**Comment by Klerith** [comment]
Currently, we are in version **3.1.2** I will check all the other issues to see what else can be added.

You will be able to specify a callback that is going to fire when the animation finishes; also, if you go backwards in the animation when it finishes, you will be notified as well.

<img width="363" alt="Screenshot 2024-01-24 at 8 35 44 PM" src="https://github.com/Klerith/animate_do_package/assets/3438503/bce6c28f-2698-412c-afdf-b006a774f84d">
.

**Comment by erperejildo** [comment]
I'm also in `animate_do: ^3.1.2` but `onFinish `is not there:
![image](https://github.com/Klerith/animate_do_package/assets/5629919/b7addfb5-849c-437c-89f8-6f64c304659c)

```
The named parameter 'onFinish' isn't defined.
Try correcting the name to an existing named parameter's name, or defining a named parameter with the name 'onFinish'.
```

**Comment by Klerith** [comment]
onFinish is not there yet; that's a sneak peek at how it will work.

**Comment by erperejildo** [comment]
> onFinish is not there yet; that's a sneak peek at how it will work.

ah sorry. I saw the screenshot and I thought it was already there.

Yeah, it looks good

**Comment by Klerith** [comment]
Published on  V3.2.0!

It should be there now, @erperejildo. Feel free to reopen the issue if that is not working for you.
